doc/src/manual/juliaup.md

```diff
@@ -0,0 +1,12 @@
+# [Juliaup](@id man-juliaup)
+
+Juliaup is the official installer for Julia, enabling you to easily update and switch between Julia versions.

```

Comment:
Should this file really be here? It seems to me it would make more sense to have a dedicated Juliaup manual, part of the Juliaup repository https://github.com/JuliaLang/juliaup and then link to that?

Reply:
Yeah that makes sense to me.

Reply:
I think from a user point of view I _really_ dislike it when documentation is spread-out, e.g. the fact that the Pkg.jl docs aren't just in the Julia manual in my mind is really not user friendly. Sooo, I think it would be really much better if the docs are just part of the Julia manual, way easier to find for users. BUT, I'm going to remove this for now, so that we can at least merge the first part of this PR here.